src/valkey-benchmark.c

```diff
@@ -1552,7 +1558,12 @@ int parseOptions(int argc, char **argv) {
         "                    on the command line.\n"
         " -I                 Idle mode. Just open N idle connections and wait.\n"
         " -x                 Read last argument from STDIN.\n"
-        " --seed <num>       Set the seed for random number generator. Default seed is based on time.\n",
+        " --seed <num>       Set the seed for random number generator. Default seed is based on time.\n"
+        " --num-functions-in-script <num> Sets the number of functions present in the Lua script\n"
+        "                                 that is loaded when running the 'function_load' test.\n"
+        "                                 (default 10).\n"
```

Comment:
The word "script" is used only for EVAL-related scripts in the docs and command names (like SCRIPT LOAD).

Library (or lib) might be the right word.